docs/source/scenarios/supply_chain.rst

```diff
@@ -0,0 +1,225 @@
+
+Supply Chain Management
+=======================
+
+The Supply Chain scenario simulates a common problem of products' manufacture, storage, and transportation. In this simulated system, products are produced by suppliers, stored in warehouses, and then sold by retailers. Multiple vehicles are running around the "world" to deliver products and materials. We need to design a strategy to control the production and transportation according to the dynamic market demand so that the entire system can operate efficiently and benignly.
+
+Resource Flow

```

Comment:
Since I had also read the BE code before, I am not able to judge how could we explain/introduce this scenario better. We can have more discussion about it.